pkg/apis/core/validation/validation.go

```diff
@@ -36,7 +36,7 @@ import (
 	"k8s.io/apimachinery/pkg/api/resource"
 	apimachineryvalidation "k8s.io/apimachinery/pkg/api/validation"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	unversionedvalidation "k8s.io/apimachinery/pkg/apis/meta/v1/validation"
+	metavalidation "k8s.io/apimachinery/pkg/apis/meta/v1/validation"
```

Comment:
Please avoid unrelated diffs to make review easier.

Reply:
Done.